rpm/SPECS/iotajson.spec

```
@@ -169,6 +169,20 @@ fi
 %{_install_dir}
 
 %changelog
+* Fri Apr 29 2022 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 1.22.0
+- Add: apply expression and payload transformations to commands (#634, #627)
+- Fix: ensure command QoS for MQTT is an integer
+- Fix: ensure mqtt client_id is unique between reconnections to avoid reconnection loop (#650)
+- Fix: bad mqtt measure is progressing as a multiple measure after be procesed as hex string (#642)
+- Fix: search device and group for a command using entity type when provided to proper match (iota-node-lib#1211)
+- Fix: replace git:// to https:// url for npm dependences
+- Remove: obsolete iotaJsonTester binary
+- Upgrade NodeJS version from 12 to 14 in Dockerfile
+- Upgrade moment dep from 2.27.0 to 2.29.2 due to security vulnerability (CVE-2022-24785)
+- Upgrade async dep from 2.6.1 to 2.6.4 due to security vulnerability (CWE-1321)
+- Upgrade MQTT dep from 3.0.0 to 4.3.7
+
```

```suggestion
- Upgrade iotagent-node-lib dependency from 2.19.0 to 2.21.0
- Upgrade moment dep from 2.27.0 to 2.29.2 due to security vulnerability (CVE-2022-24785)
- Upgrade async dep from 2.6.1 to 2.6.4 due to security vulnerability (CWE-1321)
- Upgrade MQTT dep from 3.0.0 to 4.3.7
- Upgrade NodeJS version from 12 to 14 in Dockerfile
```

Note we step two iotagent-node-lib versions, given 2.20.0 was created for the 1.21.1 hotfix